python-avd/pyavd/_eos_designs/structured_config/network_services/ip_access_lists.py

```diff
@@ -65,6 +60,16 @@ def _acl_internet_exit_direct(self: AvdStructuredConfigNetworkServicesProtocol)
 
             self.structured_config.ip_access_lists.append(acl)
 
+    def _set_zscaler_ie_policy_acl(self: AvdStructuredConfigNetworkServicesProtocol) -> None:
+        """Configure an IP access list for the Zscaler Internet policy."""
+        acls = self._acl_internet_exit_user_defined("zscaler")
```

Comment:
This probably overlaps with other comments, but I think this should be called with the ACL as argument instead. It still feels like the code is focused around "pulling" data based on output, instead of pushing/setting output while parsing the inputs.

Reply:
these ACLs are used for nat profile.